src/OpenTelemetry.Exporter.Prometheus.HttpListener/Internal/PrometheusMetric.cs

```diff
@@ -53,20 +75,25 @@ public PrometheusMetric(string name, string unit, PrometheusType type, bool disa
             sanitizedName += "_total";
         }
 
-        // Special case: Converting "1" to "ratio".
-        // https://github.com/open-telemetry/opentelemetry-specification/blob/b2f923fb1650dde1f061507908b834035506a796/specification/compatibility/prometheus_and_openmetrics.md#L239
-        if (type == PrometheusType.Gauge && unit == "1" && !sanitizedName.Contains("ratio"))
+        // For counters requested using OpenMetrics format, the MetricFamily name MUST be suffixed with '_total', regardless of the setting to disable the 'total' suffix.
+        // https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#counter-1
+        if (type == PrometheusType.Counter && !openMetricsName.EndsWith("_total"))
         {
-            sanitizedName += "_ratio";
+            openMetricsName += "_total";
```

Comment:
In the OpenMetrics spec, `_total` suffix is mandatory, whereas in the OTEL spec it can be optional via configuration. Not sure how to rationalize the 2 specifications, if someone can provide some input? In the Java version of OTEL, there is no option to disable the suffix, so the `_total` suffix is always added.

Reply:
From their site, as far as I can tell, OpenMetrics spec is `released`. It also hasn't been modified in the last 2 years, so I'd say it's pretty solidified at this point? Not an expert though.